tests/test_acm/test_acm.py

```diff
@@ -38,13 +38,24 @@ def test_import_certificate():
     client = boto3.client("acm", region_name="eu-central-1")
 
     resp = client.import_certificate(
-        Certificate=SERVER_CRT, PrivateKey=SERVER_KEY, CertificateChain=CA_CRT
+        Certificate=SERVER_CRT,
```

Comment:
Would you mind creating a new test with the tags? Would be good to keep the existing test as is, to verify that it still works without passing in tags

Reply:
Sounds right.  :+1:   
I have added few more new tests and better tag handling.